src/lib/babel-custom.js

```diff
@@ -7,6 +7,16 @@ const ESMODULES_TARGET = {
 	esmodules: true,
 };
 
+// silence Babel 7.13+ loose-with-assumptions warnings
```

Comment:
why do we want to silence this?

Reply:
Great question, will look into it & improve the comment (or remove this entirely) as I don't quite remember myself.

Reply:
Thanks for the mention! Indeed, seems like a silly hack I inserted when testing and forgot to clean up/fix.